libs/blocks/merch-card/merch-card.js

```diff
@@ -0,0 +1,205 @@
+/* eslint-disable prefer-destructuring */
+import { decorateButtons } from '../../utils/decorate.js';
+import { loadStyle, getConfig, createTag } from '../../utils/utils.js';
+import { addBackgroundImg, addWrapper, addFooter } from '../card/cardUtils.js';
+import { decorateLinkAnalytics } from '../../martech/attributes.js';
+
+const {
+  miloLibs,
+  codeRoot,
+} = getConfig();
+const base = miloLibs || codeRoot;
+const SECURE_TRANSACTION_IMG = `<img class="secure-transaction-icon-img" loading="lazy" alt="secure-transaction-icon icon" src="${base}/blocks/merch-card/img/secure-transaction-icon.svg" height="12" width="15">`;
```

Comment:
```suggestion
```
We can remove this and the `/img/` directory and reference author-able svgs from libs. See the css suggestion below.